.github/workflows/build-release.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  build-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: 'npm'
+      - name: Install
+        run: make install
```

Comment:
Avoid `make` as it's likely that Windows users which just installed `node` won't have it in their path.